.vscode/launch.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
I like to optimize my projects for a specific xplat IDE, which is VS Code for me--it provides a happy path for contributors. However, you are the maintainer so would you be OK adding the configuration to the CONTRIBUTING readme? That provides a copy/paste way for contributors who want to use VS Code at least.